source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -976,11 +977,12 @@ export default class WalletSendForm extends Component<Props, State> {
             assetsAmounts={this.selectedAssetsAmounts}
             amount={adaAmount.toFormat(currencyMaxFractionalDigits)}
             amountToNaturalUnits={formattedAmountToNaturalUnits}
-            totalAmount={total}
+            totalAmount={getTransactionFeeCalculated(adaAmount, transactionFee)}
             transactionFee={fees}
             hwDeviceStatus={hwDeviceStatus}
             isHardwareWallet={isHardwareWallet}
             onExternalLinkClick={onExternalLinkClick}
+            currencyMaxFractionalDigits={currencyMaxFractionalDigits}
```

Comment:
@danielmain perhaps instead of passing on the `currencyMaxFractionalDigits` we could rather pass the `formattedTotalAmount` and do the formatting here?

Reply:
Is this commet ok for you? `// Minimal balance amount of ADA`